<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Método de Bisección</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #e1e1e1;</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="btnCalcular">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 85, 255);
color: #fff;
font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Calcular Raíz</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(176, 176, 176);</string>
   </property>
   <property name="title">
    <string>Intervalos</string>
   </property>
   <widget class="QLineEdit" name="txtInferior">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
color: #000;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSuperior">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
color: #000;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Inferior</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Superior</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(157, 157, 157)</string>
   </property>
   <property name="title">
    <string>Ecuación</string>
   </property>
   <widget class="QPlainTextEdit" name="txtEcuacion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
color: #000;
font-size: 14px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>10</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color:rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGraficar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>341</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);
color: #fff;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>Graficar Función</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>245</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnExponente">
       <property name="text">
        <string>x^2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnRaiz">
       <property name="text">
        <string>sqrt( x )</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnParentesis">
       <property name="text">
        <string>( )</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnPor">
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnCos">
       <property name="text">
        <string>cos( x )</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnSen">
       <property name="text">
        <string>sin( x )</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnTan">
       <property name="text">
        <string>tan( x )</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnEntre">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnMas">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnMenos">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnPi">
       <property name="text">
        <string>π</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnEF">
       <property name="text">
        <string>e^x</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnLog">
       <property name="text">
        <string>log( x )</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnLn">
       <property name="text">
        <string>ln( x )</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="txtEcuacionR">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>601</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: #fff;
color: #000;
font-size: 14px;
text-align: relative;
margin-left: 10px;</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>360</y>
     <width>120</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(150, 150, 150);</string>
   </property>
   <property name="title">
    <string>Tolerancia</string>
   </property>
   <widget class="QLineEdit" name="txtTolerancia">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fff;
color: #000;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnLimpiar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: #fff;
font-size: 14px</string>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtEcuacion</tabstop>
  <tabstop>btnGraficar</tabstop>
  <tabstop>txtInferior</tabstop>
  <tabstop>txtSuperior</tabstop>
  <tabstop>txtTolerancia</tabstop>
  <tabstop>btnCalcular</tabstop>
  <tabstop>btnLimpiar</tabstop>
  <tabstop>btnCerrar</tabstop>
  <tabstop>btnExponente</tabstop>
  <tabstop>btnRaiz</tabstop>
  <tabstop>btnParentesis</tabstop>
  <tabstop>btnPor</tabstop>
  <tabstop>btnCos</tabstop>
  <tabstop>btnSen</tabstop>
  <tabstop>btnTan</tabstop>
  <tabstop>btnEntre</tabstop>
  <tabstop>btnMas</tabstop>
  <tabstop>btnMenos</tabstop>
  <tabstop>btnX</tabstop>
  <tabstop>btnPi</tabstop>
  <tabstop>btnEF</tabstop>
  <tabstop>txtEcuacionR</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
